256

```diff
@@ -0,0 +1,3 @@
+--2021-05-04 15:44:10--  http://shasum/
```

Comment:
This looks like a spurious file that shouldn't be included.

Reply:
You're right.
File removed